lib/onix/publishing_detail.rb

```diff
@@ -152,6 +152,15 @@ def imprint
       end
     end
 
+    def print_publication_date
+      pub=@publishing_dates.select{|pd| pd.role.human=="PublicationDateOfPrintCounterpart"}.first
+      if pub
+        pub.date.date
+      else
+        nil
+      end
```

Comment:
Ce `else` me semble superflu.
```suggestion
      return pub.date.date if pub
```